### ISSUE TITLE
already initialized constant Chef::Knife::Bootstrap

### ISSUE THREAD

**Issue by scalp42** [chef/chef-workstation#395]
## Description

```
$> knife status
/opt/chef-workstation/embedded/lib/ruby/gems/2.6.0/gems/chef-15.0.300/lib/chef/knife/bootstrap.rb:31: warning: already initialized constant Chef::Knife::Bootstrap::SUPPORTED_CONNECTION_PROTOCOLS
/opt/chef-workstation/embedded/lib/ruby/gems/2.6.0/gems/chef-15.0.300/lib/chef/knife/bootstrap.rb:31: warning: previous definition of SUPPORTED_CONNECTION_PROTOCOLS was here
/opt/chef-workstation/embedded/lib/ruby/gems/2.6.0/gems/chef-15.0.300/lib/chef/knife/bootstrap.rb:32: warning: already initialized constant Chef::Knife::Bootstrap::WINRM_AUTH_PROTOCOL_LIST
/opt/chef-workstation/embedded/lib/ruby/gems/2.6.0/gems/chef-15.0.300/lib/chef/knife/bootstrap.rb:32: warning: previous definition of WINRM_AUTH_PROTOCOL_LIST was here
/opt/chef-workstation/embedded/lib/ruby/gems/2.6.0/gems/chef-15.0.300/lib/chef/knife/bootstrap.rb:350: warning: already initialized constant Chef::Knife::Bootstrap::DEPRECATED_FLAGS
/opt/chef-workstation/embedded/lib/ruby/gems/2.6.0/gems/chef-15.0.300/lib/chef/knife/bootstrap.rb:350: warning: previous definition of DEPRECATED_FLAGS was here
48 minutes ago, nomad-server-i-0ccd74ab565a3962d, ubuntu 18.04.
48 minutes ago, noop-i-01876a1a82850e96d, ubuntu 18.04.
48 minutes ago, consul-i-0df507a1c352f6b6a, ubuntu 18.04.
48 minutes ago, consul-i-03a49ce0a22bc6316, ubuntu 18.04.
48 minutes ago, janitor-i-0223282baf0873505, ubuntu 18.04.
```

## Chef Workstation Version
```
$> chef --version
Chef Workstation: 0.4.2
  chef-run: 0.3.0
  chef-client: 15.0.300
  delivery-cli: 0.0.52 (9d07501a3b347cc687c902319d23dc32dd5fa621)
  berks: 7.0.8
  test-kitchen: 2.2.5
  inspec: 4.3.2
```

## Platform Version

```
MacOS 10.14.5
```

**Comment by foobarbam** [comment]
ditto on Redhat 7.6 - same versions as above.

**Comment by tyler-ball** [comment]
Thanks for the report y'all! This is a known issue with Chef 15.0 - we changed all the underlying code for `knife bootstrap` which is where those warnings are coming from. We are working on fixes for these to be released in Chef 15.1, and will pull that into a Chef Workstation release afterwords.

**Comment by tas50** [comment]
Closing this out since we totally rewrote this code and haven't heard back
